src/Parallel/Parallel.jl

```diff
@@ -1,12 +1,13 @@
 module Parallel
 
 using LightGraphs
-using LightGraphs: sample, AbstractPathState, JohnsonState, BellmanFordState, FloydWarshallState
+using LightGraphs:
+    sample, AbstractPathState, JohnsonState, BellmanFordState, FloydWarshallState
```

Comment:
why not one per line? Don't get me wrong, I like this. Perhaps we have a rule that says if your using/import is < 100 chars, a single line will work; otherwise, one per line.